### PR TITLE
add doc for refreshData

### PR DESCRIPTION
**Pull request by cojhal**

### CODE REVIEW

File: documentation/docs/api/ReportHandle.md [hunk 1]
```diff
@@ -117,7 +117,11 @@ sasReport.getReportHandle().then((reportHandle) => {
...
     // Open the PDF in a new window
-    window.open(pdfUrl, '_blank');
+    window.open(pdfUrl, "_blank");
   });
 });
 ```
+
+### refreshData(): void
+
+Refreshes the data for all objects in the report.
```
[the remark this answers is not in the document]
Comment (cojhal) [reply documentation/docs/api/ReportHandle.md]: Thank you. The latest commit uses the new wording.

[the remark this answers is not in the document]
Comment (joylashley) [reply documentation/docs/api/ReportHandle.md]: Thanks, @cojhal! That was quick.

**Comment by cojhal** [comment]
@joylashley, I added doc for ObjectHandle to this review, which provides the ability to target a specific report object when refreshing data.

File: documentation/docs/api/ObjectHandle.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+id: ObjectHandle
+title: ObjectHandle
+---
+
+An `ObjectHandle` is used to perform actions on a single object in open report.
```
Comment (brmorr) [inline documentation/docs/api/ObjectHandle.md:6]: Should be "in an open report"?

Comment (cojhal) [reply documentation/docs/api/ObjectHandle.md]: Yes, thank you.

**Comment by cojhal** [comment]
> @cojhal is this ready to merge?

Yes, unless someone has more feedback. All the comments so far have been settled.

**Pull request merged**